### PR TITLE
dev-util/idea-community: version bump

### PR DESCRIPTION
**Pull request by fancywriter**

### CODE REVIEW

File: dev-util/idea-community/idea-community-2016.1.1.ebuild [hunk 1]
```diff
@@ -9,14 +9,14 @@ SLOT="$(get_major_version)"
 RDEPEND=">=virtual/jdk-1.6"
 
 MY_PN="idea"
-MY_PV="$(get_version_component_range 4-6)"
+MY_PV="145.597.3"
```
Comment (mrueg) [inline dev-util/idea-community/idea-community-2016.1.1.ebuild]: Please don't hardcode it into the ebuild

Comment (p-kozlov) [inline dev-util/idea-community/idea-community-2016.1.1.ebuild]: Good point. But I have no idea how to identify it before unpacking the archive.

**Comment by p-kozlov** [comment]
@mrueg please check this again

**Comment by mrueg** [comment]
Please use the version that is available in tree.

**Pull request closed without merging**